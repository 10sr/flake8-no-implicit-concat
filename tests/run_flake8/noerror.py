a = "aaa"
b = "bbb" + "ccc"
d = """ddd
eee
fff""" + "ggg"
l = ["hhh",
     "iii" +
     "jjj"]
noqa = "k" "l"  # noqa: NIC001
noqa = ("k"  # noqa: NIC002
        "l")
print("abc" + "def", "ghi")
