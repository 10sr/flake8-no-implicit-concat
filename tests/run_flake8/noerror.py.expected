./noerror.py:10:12: NIC002 Implicitly concatenated string literals over multiple lines
